src/components/feedbackModal.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from 'react';
+import { useContext } from 'react';
+import UserDataContext from '../context/UserDataContext/UserDataContext';
+import { Transition } from '@headlessui/react';
+
+const UserAuthButton = props => {
+  const { firebaseUser, signIn, signOut } = useContext(UserDataContext);
+  return (
+    <button {...props} onClick={() => (firebaseUser ? signOut() : signIn())}>
+      {firebaseUser ? 'Logout' : 'Login'}
+    </button>
+  );
+};
+
+export default function FeedbackModal({ isOpen, onClose }) {
```

Comment:
Wait the code for this file is the settings modal right?

Reply:
Pretty sure

Reply:
then why is it in feedbackModel.tsx :P (also is this file even used?)

Reply:
@thecodingwizard Honestly I forget what my logic was when coding stuff up like two days after I do it (if you showed me code I wrote a week ago I genuinely would not know it was mine lol) but I would keep it just in case I used something in there.

Reply:
I think we should at least _try_ to write maintainable code ;)